ui/webui/test/helpers/common_helpers.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2022 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; If not, see <http://www.gnu.org/licenses/>.
+
+
+class CommonHelpers():
+    # pylint: disable=no-member
+
+    def begin_installation(self):
+        self.browser.click("button:contains('Begin installation')")
```

Comment:
Since all these helper functions use the browser object anyway, wouldn't be easier to create wrappers for the browser? So instead of using methods of `CommonHelpers`, we could create `i = Installer(b)` and `l = LanguageScreen(b)` instead of `LanguageHelpers`. What do you think?

Reply:
I like this approach.

Reply:
However, these class instances will need to be created once per test.

Reply:
> However, these class instances will need to be created once per test. It will slightly increase the amount of code, but it's totally fine from my side.

I think that's fine.

Reply:
oh let me adapt also the class names, according to the suggestion.